src/datavalues/data_value_operator.rs

```diff
@@ -65,10 +65,10 @@ pub enum DataValueArithmeticOperator {
 impl std::fmt::Display for DataValueArithmeticOperator {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         let display = match &self {
-            DataValueArithmeticOperator::Add => "+",
-            DataValueArithmeticOperator::Sub => "-",
-            DataValueArithmeticOperator::Mul => "*",
-            DataValueArithmeticOperator::Div => "/",
+            DataValueArithmeticOperator::Add => "plus",
+            DataValueArithmeticOperator::Sub => "minus",
```

Comment:
Maybe we can also rename the name to be consistent.  

`DataValueArithmeticOperator::Add` => `DataValueArithmeticOperator::Plus`

Reply:
done